near/omni-bridge/src/lib.rs

```diff
@@ -225,7 +225,7 @@ impl FungibleTokenReceiver for Contract {
         if self.deployed_tokens.contains(&token_id) {
             ext_token::ext(token_id.clone())
                 .with_static_gas(BURN_TOKEN_GAS)
-                .burn(amount);
+                .burn((amount.0 - transfer_message.fee.fee.0).into());
```

Comment:
Nitpick: maybe we should use `parsed_msg`, since for `require_storage_balance` we use it as well and it removes one level of structure nesting?
```suggestion
                .burn((amount.0 - parsed_msg.fee.0).into());
```

Reply:
I think it doesn’t matter at all; it’s the same value. I chose transfer_message because we use it for the check above.

Reply:
Okay, that makes sense